src/main/java/frc/robot/Robot.java

```diff
@@ -52,21 +52,29 @@ public void autonomousInit() {
 
 	@Override
 	public void autonomousPeriodic() {
-		everybotIntake.update(null);
```

Comment:
So is everybotIntake inactive during autonomous?

Reply:
for now were not doing deposit for auto so no mech in that